internal/testing/processor/testdata/github-review.json

```diff
@@ -0,0 +1,13 @@
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [
+      {
+        "name": "git://github.com/kubernetes/kubernetes", 
+        "digest": {"sha1": "5835544ca568b757a8ecae5c153f317e5736700e"}
+      }
+  ],
+  "predicateType": "http://in-toto.io/attestation/review",
+  "predicate": {
+    "full_review": "https://github.com/kubernetes/kubernetes/pull/112078#pullrequestreview-1088153270"
```

Comment:
i'm not quite sure what the `full_review` here is trying to say ... Maybe we can change this to just a certification attestation "http://in-toto.io/attestation/certify`, and have things like who the certifier is, etc.

Reply:
Changed this to be `Certify` and added additional fields. Good to get feedback from @mlieberman85 @SantiagoTorres

Reply:
Does it make sense to just make it clear this is just like a test certification type or something? Or like "alpha" attestation or however we want to structure it. I just want to make sure it's clear that this isn't a widely accepted predicate yet.